Makefile

```diff
@@ -45,6 +45,13 @@ ifeq ($(OS),FreeBSD)
 	install -m 644 doc/man/httpdirfs.1.gz \
 		$(DESTDIR)$(prefix)/man/man1/httpdirfs.1.gz
 endif
+ifeq ($(OS),Darwin)
+	install -m 755 httpdirfs \
+		$(DESTDIR)$(prefix)/bin/httpdirfs
+	install -d $(DESTDIR)$(prefix)/share/man/man1
```

Comment:
What does this line do? Is it necessary?

Reply:
My local system's man page says

           -d, --directory
              treat all arguments as directory names; create all components of the specified directories

Does that man on macOS, that directory does not exist? Does the man page work under macOS with your proposed changes?

Reply:
`install` on macOS does not create target directories and they have to be created in a separate step with `-d`. Now that I come to think of it, the `bin` directory needs similar treatment too.

Reply:
`install` on macOS does not support the `-D` flag and fails if the target directory does not exist so one has to create it in a separate step with `install -d`. I've tested that the man page works and Homebrew installs uncompressed manages.

Reply:
Cool, I have merged your changes. :)

Reply:
Thanks! Hopefully that makes your excellent tool available to more people :)

Reply:
Thanks for providing the patch! :)

Reply:
Regarding your question - `/usr/local` is empty on newly installed macOS and you need to populate it from scratch. Homebrew does so and installs its packages there on Intel macs but have switched to `/opt/homebrew` for native Apple Silicon (M1, ARM) packages.

Reply:
Okay, now I have one more question. Your Makefile does not specify ``prefix`` nor ``DESTDIR``, are they set by Homebrew? If not, should they be added in the Makefile?

Reply:
`prefix` gets a default value on line 24: `prefix ?= /usr/local`. `DESTDIR` is empty by default. I guess its purpose is to facilitate building packages in fakeroot environments. Homebrew is generally a package manager, but it can also build software from source if you write a recipe for it. Recipes are written in Ruby and they can set all manner of variables, but Ruby isn't something I'm fluent in so I hope someone else could write a recipe and submit it to Homebrew :)

Reply:
Ah ok, I am not familiar with macOS. So this Makefile and Homebrew are two separate things then, I suppose? I was slightly concerned about your patch doesn't work for M1 Mac.

Reply:
It works perfectly on M1 Macs. I actually developed the patch on an M1 Mac mini and also tested it on an Intel MacBook. If a Homebrew recipe is ever written, it should take care of setting `prefix` accordingly when calling `make` while building the package.

Reply:
Okay, cool. :)